ProjectLighthouse/Helpers/EmailHelper.cs

```diff
@@ -59,7 +61,8 @@ public static async Task SendPasswordResetEmail(DatabaseContext database, IMailS
         string messageBody = $"Hello, {user.Username}.\n\n" +
                              "A request to reset your account's password was issued. If this wasn't you, this can probably be ignored.\n\n" +
                              $"If this was you, your {ServerConfiguration.Instance.Customization.ServerName} password can be reset at the following link:\n" +
-                             $"{ServerConfiguration.Instance.ExternalUrl}/passwordReset?token={token.ResetToken}";
+                             $"{ServerConfiguration.Instance.ExternalUrl}/passwordReset?token={token.ResetToken}\n\n" + 
+                             "This link will expire in one hour";
```

Comment:
Just noticed this, should this not say 24 hours instead of 1 hour?

Reply:
I fixed this yesterday but apparently forgot to push the commit